src/resourcetiming-compression.js

```diff
@@ -1063,6 +1083,25 @@
             } else {
                 results[finalUrl] = data;
             }
+
+            if (e.hasOwnProperty("_data")) {
+                var namespacedData = "";
```

Comment:
Looks like we already have decompression tests for namespaced data, can you add compression tests too?

Reply:
I'll try.